data/actions/actions.xml

```diff
@@ -132,6 +132,11 @@
 	<action itemid="6120" script="other/spellbook.lua" />
 	<action fromid="8900" toid="8904" script="other/spellbook.lua" />
 	<action itemid="8918" script="other/spellbook.lua" />
+	<action fromid="22422" toid="22424" script="other/spellbook.lua" />
+	<action itemid="18401" script="other/spellbook.lua" />
+	<action itemid="16112" script="other/spellbook.lua" />
+	<action itemid="15411" script="other/spellbook.lua" />
```

Comment:
Depth scutum should not work as a spellbook. Could you also re-order them in ascending order of the item IDs? Please squash your commits when you're done.